metadata-db-app/int-test/cmr/metadata_db/int_test/utility.clj

```diff
@@ -233,26 +233,28 @@
 
 (defn- save-concept-core
   "Fundamental save operation"
-  [concept]
-  (let [response (client/post (concepts-url)
-                              {:body (json/generate-string concept)
-                               :content-type :json
-                               :accept :json
-                               :throw-exceptions false
-                               :connection-manager (conn-mgr)})
-        status (:status response)
-        body (json/decode (:body response) true)
-        {:keys [revision-id concept-id errors variable-association]} body]
-    (if (= :variable (:concept-type concept))
-      {:status status
-       :revision-id revision-id
-       :concept-id concept-id
-       :errors errors
-       :variable-association variable-association}
-      {:status status
-       :revision-id revision-id
-       :concept-id concept-id
-       :errors errors})))
+  ([concept] (save-concept-core concept nil))
+  ([concept headers]
+   (let [response (client/post (concepts-url)
+                               {:body (json/generate-string concept)
+                                :content-type :json
+                                :accept :json
+                                :headers (when headers headers)
```

Comment:
Don't feel `(when headers headers)` is necessary. How is it different from `headers`?